drivers/docker/driver_test.go

```diff
@@ -651,6 +651,11 @@ func TestDockerDriver_Start_KillTimeout(t *testing.T) {
 		t.Parallel()
 	}
 	testutil.DockerCompatible(t)
+
+	if runtime.GOOS == "windows" {
+		t.Skip("Windows Docker does not support SIGUSR1")
```

Comment:
We should write this test in a way that accounts for Windows.  We can change the signal used, or use a process that ignores SIGINT/SIGTERM.